pkg/provider/azure_loadbalancer.go

```diff
@@ -1961,6 +1980,17 @@ func (az *Cloud) accommodateNodesByNodeSelector(
 	return nil
 }
 
+func checkLBInList(lbs *[]network.LoadBalancer, lbConfigName string) bool {
```

Comment:
rename to isLBinList for clarity
And could you add some comments for what does lbConfigName mean here?

Reply:
done